lib/shared/types/tsconfig.json

```diff
@@ -1,7 +1,9 @@
 {
   "extends": "../../../tsconfig.base.json",
   "compilerOptions": {
-    "module": "CommonJS",
+    "target": "es2015",
```

Comment:
can we target a more recent version for this?

Reply:
Think so, I'll give it a shot.